power_spec.adoc

```diff
@@ -0,0 +1,161 @@
+Carole-Jean Wu (carolejeanwu@fb.com)
+
+== Goals
+
+This draft document describes the power measurement procedure for MLPerf Inference
+Suite v0.5 only. The main goals are as follows:
+
+* Initial v0.5 power measurement methodology for wall-powered systems that emphasizes minimal effort
+* Identify longer term goals that are candidates for later iterations (e.g., v1.0)
+* Re-use existing SPECPower recommended methodology as much as possible.
+* Balance reproducibility, accuracy, and fairness of power methodology against burden imposed on submitters
+* Measure the entire system, to include any component that contributes to the performance score (accelerator card, host, networking).
+
+== Requirement and Scope
+
+Submitting power is _optional_ in MLPerf Inference Suite v0.5. Submitters who
+wish to submit power must follow this spec. This power specification covers
+wall-powered systems only. This power spec is currently only applicable to the
+Inference benchmark. We expect future versions will support additional features
+to enable its application for the MLPerf Training Suite.
+
+== Power management
+
+The goal of this benchmark is to mimic real-world usage scenarios as much
+as possible, and enable demonstrating of the benefits of realistic power
+management. Therefore, we require any power management system to:
+
+* Be qualified for use appropriate for the
+submission type (e.g., a generally available system must use
+software/firmware qualified for general availability and shipping
+with the platform)
+* Include no benchmark or benchmarking-specific modifications
+* Changes in power management behavior must not have manual intervention or
+be aware of the benchmark
+
+== Wall-Powered System Under Test
+
+The System under Test (SUT) includes any component that contributes to the
+measured performance of the MLPerf benchmark. For example, an
+accelerator based system should also include the host node,
+networking/switches for distributed systems, number and type of cooling systems,
+and hard disks. If the system uses shared resources with multiple servers, the
+shared resources must be included in the SUT.
+
+The cooling method must be described (e.g., air-cooling, liquid cooling).
+
+Figure 1 below illustrates details of the MLPerf power measurement configuration.
+Note that in contrast to SPECpower, the temperature sensor is not required, and
+the collection system is not coupled to the SUT and driving/controlling the SUT.
+All power delivered to the SUT must arrive through the power meter.
+
+DC-powered systems are also supported with an appropriate power meter, but may
+not be compared to AC-powered systems. If a wall-powered system is available in
+both an AC-powered and DC-powered configuration, we require using the
+AC-powered configuration.
+
+image::sut_power_setup.png[SUT Setup]
+
+== Wall-Powered Measurement Procedure
+
+. Connect an approved power meter from this list to the SUT.
```

Comment:
Am I missing something? "an approved power meter from this list" -- where is this list?

Reply:
https://www.spec.org/power/docs/SPECpower-Device_List.html

I don't know how to fix Hanlin's commit because I don't know how to use github, but this is the answer!

Reply:
Updated with a link to the list from David above.